### PR TITLE
Updated installation scripts (AB-42)

### PR DESCRIPTION
**Pull request by gentlecat**
*Fixes [AB-42](http://tickets.musicbrainz.org/browse/AB-42).*

### CODE REVIEW

File: admin/install_hl_extractor.sh [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
+else
+    APP_DIR=$(dirname $0)/..
+    echo "Application directory is not specified. Using default: '$APP_DIR'!"
+fi
+
 # Installing dependencies
-apt-get -y install build-essential libyaml-dev libfftw3-dev libavcodec-dev \
-    libavformat-dev python-dev python-numpy-dev python-numpy git \
-    libsamplerate0-dev libtag1-dev libqt4-dev swig pkg-config
+apt-get -y --force-yes install build-essential libyaml-dev libfftw3-dev \
```
Comment (alastair) [inline admin/install_hl_extractor.sh]:
```
       --force-yes
           Force yes; This is a dangerous option that will cause apt to
           continue without prompting if it is doing something potentially
           harmful. It should not be used except in very special situations.
```
Why do we need force yes?

Comment (gentlecat) [inline admin/install_hl_extractor.sh]: I had issues with signatures on some of these libraries when I tried to install them today. Let me try again and see if it works without `--force-yes`.

Comment (gentlecat) [inline admin/install_hl_extractor.sh]: Alright, seems to be working.

Comment (alastair) [inline admin/install_hl_extractor.sh]: ok, fine. If there's a signature error then the process should have included `apt-key add` to load signatures :)

**Comment by alastair** [comment]
fine. :ship:
